nestkernel/archiving_node.h

```diff
@@ -154,6 +190,25 @@ class Archiving_Node : public Node
 
   // spiking history needed by stdp synapses
   std::deque< histentry > history_;
+
+  /*
+   * Structural plasticity
+   */
+
+  // Time of the last update of the Calcium concentration
+  double_t Ca_t_;
```

Comment:
Comments should state the units; for times, it is probably ms, but what is it for `Ca_minus_` and `beta_Ca_`?